### PR TITLE
align cli docs

### PR DESCRIPTION
**Pull request by tomermevorach**

### CODE REVIEW

**Comment by hadasb** [comment]
Maybe we should write something about static resources in deploy's detailed-plan?
Or mention that this is a semantic based plan?

File: docs/user_guide.md [hunk 1]
```diff
@@ -364,27 +369,44 @@ Manage your environments' services
...
+### **salto diff \<from-env> \<to-env> [element-selector..]**
 
-Manage your workspace environments
+Compare two workspace environments
 
 **Arguments:**
-* `command` : The environment management command [string] [required] [choices: "create", "set", "list", "current", "delete", "rename"]
-* `name` : The name of the environment (required for create, set, delete & rename) [string]
-* `new-name` : The new name of the environment (required for rename) [string]
+* `from-env` : The environment to be align from [string] [required]
```
Comment (hadasb) [inline docs/user_guide.md:400]: from-env: The environment that serves as a baseline for the comparison.
to-env: The environment that is compared to the baseline provided by from-env.

**Comment by tomermevorach** [comment]
> Maybe we should write something about static resources in deploy's detailed-plan?
> Or mention that this is a semantic based plan?

I am afraid it would be too confusing, in order to do that you first need to explain what is a static resource which is a bad term by itself. I'd leave it as is for now

File: docs/user_guide.md (hunk 1 above)
Comment (tomermevorach) [reply docs/user_guide.md]: > from-env: The environment that serves as a baseline for the comparison.
> to-env: The environment that is compared to the baseline provided by from-env.

thx fixed both in the doc and the CLI itself :)

**Review by hadasb — APPROVED** [review]

**Pull request merged**